### PR TITLE
triangle: Added parameters to exercise placeholder

### LINKED ISSUE #647: triangle: add parameters to exercise placeholder
We would like you to help us add expected parameters in placeholder templates as described in #509 to make it easier for learners to start working on the exercise in question.

Feel free to ask any questions, we'll be glad to help!

### PR DESCRIPTION
**Pull request by rozuur**
fixes #647

### CODE REVIEW

File: exercises/triangle/triangle.py [hunk 1]
```diff
@@ -3,5 +3,8 @@ class TriangleError(Exception):
 
 
 class Triangle(object):
-    def __init__(self):
+    def __init__(self, x, y, z):
```
Comment (m-a-ge) [inline exercises/triangle/triangle.py]: Honestly, I don't like single character names. Could you change them to something more meaningful 

A good Python style guide from Google - https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming

Comment (rozuur) [reply exercises/triangle/triangle.py]: what do you propose? I have two suggestions, edge_1, edge_2, edge_3 or side_a, side_b, side_c

**Comment by m-a-ge** [comment]
@rozuur thanks for your PR

Just for future reference a guide of how to write good commit messages - https://chris.beams.io/posts/git-commit/#seven-rules

**Comment by rozuur** [comment]
Thanks @m-a-ge

**Pull request merged**